mqtt/mqttd/src/main.rs

```diff
@@ -1,3 +1,4 @@
+#![type_length_limit = "1230974"]
```

Comment:
Necessary to fix the build with these new changes. It doesn't seem to signify a problem, just that we now are exceeding a type substitution limit.
https://doc.rust-lang.org/reference/attributes/limits.html